pkg/kubectl/polymorphichelpers/mapbasedselectorforobject_test.go

```diff
@@ -19,10 +19,10 @@ package polymorphichelpers
 import (
 	"testing"
 
+	api "k8s.io/api/core/v1"
```

Comment:
`corev1` is the preferred alias, to be explicit what we're working with in the code.